.travis.yml

```diff
@@ -1,14 +1,15 @@
+sudo: false
 language: node_js
 before_install:
   - curl --location http://git.io/1OcIZA | bash -s
```

Comment:
I'm not happy including this, the control of this gist is outside of the control of this repo and it's a bit too opaque, I'd rather either leave this out or put the logic locally